### DISCUSSION TITLE
柿子 | /cn/2022/04/ichiko/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1482] [Comments]
柿子
是我耳朵里的魔法词
每当我听到一声
谢柿子
无论我身在几次元
都会被通灵进厨房
嘎嘣脆的麻花
流油的牛肉包子
外焦里软的千层葱油饼
园子里的长条鲜辣椒
还有南瓜派和提拉米苏
以及枣糕和椰蓉面包
它们都源源不断
像泉水一样，从我手头
往外冒啊冒

https://yihui.org/cn/2022/04/ichiko/

**Comment by wendyinnis** [comment]
你做饭这么厉害？

**Reply by yihui** [reply]
是啊，没想到吧，其实吹牛更厉害嘞。

**Comment by wendyinnis** [comment]
好吧柿子， 人民群众爱听你吹牛。 另外枣糕比椰蓉面包好吃

**Reply by yihui** [reply]
这一届人民群众真会捧场！将来有缘的话可以来厨房现场检验一下成果，看我如何吹破牛皮。

我觉得枣糕比椰蓉面包稍微好做一点，属于人有多大胆、糕有多大产的类型。说实话我并不是很擅长做面包，只是有一次碰巧蒙对了。

**Comment by ZaiyanHere** [comment]
牛肉包子拗口的话，就弄猪肉大葱的吧！

**Reply by yihui** [reply]
我看行。

其实我近几年开始吃素了，肉食一般都是做了别人吃，自己最多尝一下味道。我自己非常喜欢做一种油酥面混合豆渣作馅的素包子，葱油炸面粉出来的油酥很香，而我也是超级爱豆人，喜欢豆类，所以这种素包子我也很爱吃。

**Comment by ZaiyanHere** [comment]
「柿为知己者死」哈哈哈哈哈哈哈，谐音梗扣分啦！

**Reply by yihui** [reply]
扣多少分？666 吗？
